### DISCUSSION TITLE
Full Spec and Rationale

### DISCUSSION THREAD

**Discussion by Ovid** [Perl-Oshun/oshun#4] [General]
Until this library is made public, we don't have a wiki. So [Damian's gist on the topic is here](https://github.com/Ovid/heimdall/blob/main/docs/heimdall.md). It's very, very long.

**Comment by rwp0** [comment]
I think that gist should also be put in this repository as a Markdown file somewhere possibly in the archive or misc directory with the Gist URL commented on top of it just to document the source.

(Also) because: What if Damian delete it? 🙂

**Reply by Ovid** [reply]
[Done](https://github.com/Ovid/heimdall/blob/main/docs/heimdall.md). Thanks for the suggestion.
